include/spirv-tools/libspirv.hpp

```diff
@@ -180,6 +180,11 @@ class SpirvTools {
   std::unique_ptr<Impl> impl_;  // Unique pointer to implementation data.
 };
 
+// A message consumer that can be used by command line tools like spirv-opt and
+// spirv-val to display messages.
+void CLIMessageConsumer(spv_message_level_t level, const char*,
```

Comment:
Can we move this into a file in tools/utils or something similar? I don't think we want this in the public API.

Reply:
Sure.  My thought was that is make is available to other so they don't have to write their own message consumer, but I don't have strong feelings about it either way.